doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -17,6 +17,11 @@
 #### [Symfony]
 - Bumped Symfony packages to "^6.4".
 
+> [!WARNING]  
+> For [environment variable consistency purposes](https://github.com/pimcore/pimcore/issues/16638) in boostrap, please fix `public/index.php` in project root by moving `Bootstrap::bootstrap();` just above `$kernel = Bootstrap::kernel()` line instead of outside the callback function.
```

Comment:
```suggestion
> For [environment variable consistency purposes](https://github.com/pimcore/pimcore/issues/16638) in boostrap, please fix `public/index.php` in project root by moving `Bootstrap::bootstrap();` just above `$kernel = Bootstrap::kernel()` line instead of outside the closure.
```